src/current/v23.1/sso-db-console.md

```diff
@@ -6,94 +6,105 @@ docs_area: manage
+CockroachDB clusters allow users to authenticate with Single Sign-on (SSO), both to the [DB Console]({% link {{ page.version.version }}/ui-overview.md %}), and for SQL client access.
 
-CockroachDB supports SSO via [OpenID Connect (OIDC)](https://openid.net/connect/), an authentication layer built on top of OAuth 2.0.
+Cluster Single Sign-On (SSO) enables users to access the SQL interface of a CockroachDB cluster (whether provisioned on CockroachDB {{ site.data.products.cloud }} or self-hosted) with the full security of Single Sign-On (SSO), and the convenience of being able to choose from a variety of cloud-based or customer-managed identity providers (IdPs).
+
+Single sign-on (SSO) for DB Console allows a CockroachDB user to access the [DB Console]({% link {{ page.version.version }}/ui-overview.md %}) in a secure cluster via an OpenID Connect (OIDC) client and an external identity provider. When SSO is configured and enabled, the [DB Console login page]({% link {{ page.version.version }}/ui-overview.md %}#db-console-access) will display an OAuth login button in addition to the password access option.
+
+This page describes:
+
+- The login flow for users, once a cluster is SSO-enabled.
+- Configuring a CockroachDB Cluster for OIDC SSO authentication to the DB Console.
+- Provioning an OIDC client for CockroachDB Cluster SSO authentication, using GCP.
+
+Additional configuration is needed to enable SSO authentication for SQL clients, which is described here: [Cluster Single Sign-on (SSO) using a JSON web token (JWT)]({% link {{ page.version.version }}/sso-sql.md %}) 
 
 {{site.data.alerts.callout_info}}
-SSO for DB Console is available to CockroachDB {{ site.data.products.cloud }} customers.
+SSO for DB Console is available to CockroachDB {{ site.data.products.dedicated }} customers.
 
 On CockroachDB {{ site.data.products.core }} clusters, SSO for DB Console requires an [Enterprise license]({% link {{ page.version.version }}/enterprise-licensing.md %}).
 {{site.data.alerts.end}}
 
-## Requirements
+This SSO implementation uses the [authorization code grant type](https://tools.ietf.org/html/rfc6749#section-4.1) to authenticate a user.
 
-- An external OAuth 2.0 identity provider
+**Prerequisites**:
 
-## Login flow
+- **IdP:** You must have access to an OAuth 2.0 identity provider and client. The process of provisioning one with Google Cloud Platform is described here: [Provisioning an OAuth client using Google Cloud Platform (GCP)](#provision-an-oauth-client-using-google-cloud-platform-gcp)
+
+- **CockroachDB Cluster**: you must have access to one of the following:
+	1. A {{ site.data.products.core }} cluster enabled with a valid [CockroachDB Enterprise license]({% link {{ page.version.version }}/enterprise-licensing.md %}).
```

Comment:
I'd unorder the two cluster options with just a bullet instead of a number since it's either or